### PR TITLE
Add Homepage

### PR DESCRIPTION
**Pull request by newtonrd**
Casey, please review this new homepage before I just merge it. 👍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Robort
+# Robort is Awesome
```
Comment (newtonrd) [inline README.md:1]: This isn't a homepage

**Comment by newtonrd** [comment]
:shipit:

**Comment by lydonb** [comment]
:shipit:

**Comment by ehmitchell** [comment]
:shipit:

**Pull request merged**